.travis.yml

```diff
@@ -11,9 +11,9 @@ script:
   - tox
 deploy:
   - provider: pypi
-    user: theforeman
+    user: "__token__"
```

Comment:
Is this intended? Some interesting travis feature?

Reply:
pypi feature actually: https://pypi.org/help/#apitoken